testing/dub/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Mathias LANG <pro.mathias.lang@gmail.com>
+# Maintainer: Mathias LANG <pro.mathias.lang@gmail.com>
+pkgname=dub
+pkgver=1.18.0
+pkgrel=0
+pkgdesc="Package and build management system for D"
+url="http://code.dlang.org/"
+arch="x86_64"
+license="MIT"
+depends="libcurl"
```

Comment:
is this really required ? from my testing it doesn't link to  libcurl automatically so no binary uses it

Reply:
Yes. Phobos (the D standard library) does not link to `curl`, instead it loads it [at runtime](https://github.com/dlang/phobos/blob/e70d86b75d82f756ceec3b1398b77d1c1e9248bd/std/net/curl.d#L4165-L4195)

Reply:
It's possible to build `dub` without curl though (if desired) by relying on Vibe.d (see the `net` config). However, this requires one to have a dub build in the first place, so this could only be done after bootstrapping.